jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -191,6 +199,9 @@ def single_pipeline(job) {
         stage '020-install-pe'
         step020_install_pe(SKIP_PE_INSTALL, SCRIPT_DIR, server_era)
 
+        stage '025-collect-facter-data'
```

Comment:
If we think all of these steps are going to end up being the same for a single as for an individual job instance in a multipass pipeline, at some point it would seem nice, if possible, to break the calls to these steps out into a helper method - maybe parameterized to indicate whether a `stage` line should be outputted before calling the underlying step helper function?

I keep forgetting to add a step to the `single_pipeline` when doing testing of a new function against the `multipass_pipeline` -- and vice-versa.  Having the common helper that the two share might help prevent us from these unintentionally drifting / breaking.

Maybe you can't do that in a `node` block here, though?